.github/ISSUE_TEMPLATE.md

```diff
@@ -1,43 +1,43 @@
-To help us pinpoint your issue, please insert the output of the
-following commands when ran on the system with the issue:
+To help in root causing the issue, please insert the output of the
+following commands from the system with the issue:
 
 ```shell
 $ fwupdmgr --version
 **INSERT OUTPUT HERE**
 ```
 
-Note, the switch `--version` is only present since version 0.9.6. If you use an
-earlier version, please use the package manager to find out the package
-version. For example, `dpkg -l fwupd`.
-
 ```shell
 $ fwupdmgr get-devices
 **INSERT OUTPUT HERE**
 ```
 
+Please answer the following questions:
+
+- Operating system and version:
+- How did you install fwupd (examples: `apt`, `dnf`, `pacman`, `snap`, `source`):
+- Have you tried rebooting?
+- Is this a regression?
+
+<details><summary>UEFI issues</summary>
+<p>
+Please fill theses out only for UEFI specific issues:
+
 ```shell
 $ efibootmgr -v
 **INSERT OUTPUT HERE**
-**This is only required if you use the UEFI plugin**
 ```
 
 ```shell
 $ efivar -l | grep fw
 **INSERT OUTPUT HERE**
-**This is only required if you use the UEFI plugin**
 ```
 
 ```shell
 $ tree /boot
 **INSERT OUTPUT HERE**
-**This is only required if you use the UEFI plugin**
-**We're looking for any `.cap` files and the location of `fwupx64.efi`**
 ```
-
-Please answer the following questions:
-
-- Operating system and version:
-- How did you install fwupd (ex: `from source`, `pacman`, `apt-get`, etc):
-- Have you tried rebooting?
+Additional questions:
 - Are you using an NVMe disk?
 - Is secure boot enabled (only for the UEFI plugin)?
+</p>
+</details>
```

Comment:
??

Reply:
Magic markdown collapsing stuff.  Preview the markdown page in a markdown previewer (like vscode markdown preview)

Reply:
(The real reason for this PR)

Reply:
Although maybe need to change to "new" template workflow here: https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates#issue-templates .

Reply:
Yeah this should be redone with that flow, so hang tight.